src/styles/utils/variables.scss

```
@@ -0,0 +1,21 @@
+$calendar-cell-width: 100px;
+$calendar-cell-height: 100px;
```

Don't need to duplicate variables with same values
```suggestion
$day-size: 100px;
```